Strings/src/main/java/net/avalith/App.java

```diff
@@ -0,0 +1,177 @@
+package net.avalith;
+
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class App 
+{
+    // Exercise 1
+
+    public static Boolean isPalindrome(String str){
+
+        str = str.replace(" ","");
+        StringBuilder sb = new StringBuilder(str);
+
+       return sb.reverse().toString().equalsIgnoreCase(str);
+
+    }
+
+    // Exercise 2
+
+    public static Integer fromRomanToInteger(String romanNumber){
+
```

Comment:
```suggestion
        String regex = "^M{0,4}(CM|CD|D?C{0,3})(XC|XL|L?X{0,3})(IX|IV|V?I{0,3})$";
        if (!romanNumber.matches()){
            throw new StringNotRomanException();
        }
```
Pongamos este check antes, asi no creamos el `Map` si el `String` no es valido. La clase `String` tiene un metodo `matches()` que recibe un regex, te ahorras todas esas lineas de abajo